utils/count_oval_objects.py

```diff
@@ -54,29 +71,38 @@ def find_oval_objects(oval_refs):
                 definition = d
                 break
         if definition is not None:
+            extend_refs = []
             for criterion in definition.findall(".//criterion"):
                 test_ref = criterion.attrib["test_ref"]
                 tests.append(test_ref)
+            for extend_def in definition.findall(".//extend_definition"):
+                extend_ref = extend_def.attrib["definition_ref"]
+                t = get_ext_def_tests(oval_root, extend_ref)
+                tests += t
```

Comment:
As far as I know, there shouldn't be difference under the hood, but lets be consistent and use `tests.extend()`